examples/image-carousel/elements/Image.js

```diff
@@ -0,0 +1,52 @@
+const Tram = require('../../../tram-one')
+
+const html = Tram.html()
+
+const divStyle = `
+  width : 100%;
+`
+
+const imgStyle = `
+  width : 100%;
+  height : 400px;
+  margin : auto;
+  display : block;
+`
+
+const textCenter = `
+  text-align : center;
+`
+
+const textLeft = `
+  float : left;
+  padding : 5px 7px;
+  background-color : black;
+  color : white;
+  cursor : pointer;
+  margin-bottom : 10px;
+`
+
+const textRight = `
+  float : right;
+  padding : 5px 7px;
+  background-color : black;
+  color : white;
+  cursor : pointer;
+  margin-bottom ; 10px;
+`
+
+module.exports = (attrs, children) => {
+
+  const img = attrs.tiles[attrs.selected];
+
+  return html`
+    <div style=${divStyle}>   
+        <img style=${imgStyle} src=${img.image} alt=${img.alt}/>
+        <h3 style=${textCenter}>${img.text}</h3>
+      <h3>
+        <span onclick=${attrs.onPreviousSelect} style=${textLeft}>Previous</span>
+        <span onclick=${attrs.onNextImage} style=${textRight}>Next</span>
```

Comment:
Is there a reason you are not using a button here? 
If you keep these as spans make sure that you use  `user-select: none;` in your style as there is a weird highlight text issue if you click the span multiple times.

Reply:
Buttons can have odd styling, so I'm actually okay with a span here. `user-select: none;` would be cool, but I'm not that worried about it.